statsmodels/tsa/vecm/vecm.py

```diff
@@ -0,0 +1,2146 @@
+
+        Parameters
+        ----------
+        caused : int or str or sequence of int or str
+            If int or str, test whether the variable specified via this index
+            (int) or name (str) is Granger-caused by the variable(s) specified
+            by `causing`.
+            If a sequence of int or str, test whether the corresponding
+            variables are Granger-caused by the variable(s) specified
+            by `causing`.
+        causing : int or str or sequence of int or str or None, default: None
+            If int or str, test whether the variable specified via this index
+            (int) or name (str) is Granger-causing the variable(s) specified by
+            `caused`.
+            If a sequence of int or str, test whether the corresponding
+            variables are Granger-causing the variable(s) specified by
+            `caused`.
+            If None, `causing` is assumed to be the complement of `caused`.
+        signif : float between 0 and 1, default 5 %
+            Significance level for computing critical values for test,
+            defaulting to standard 0.95 level.
+        verbose : bool
+            If True, print a table with the results.
+
+        Returns
+        -------
+        results : dict
+            A dict holding the test's results. The dict's keys are:
+            * "statistic" : float
+                The claculated test statistic.
+            * "crit_value" : float
+                The critical value of the F-distribution.
+            * "pvalue" : float
+                The p-value corresponding to the test statistic.
+            * "df" : float
+                The degrees of freedom of the F-distribution.
+            * "conclusion" : str {"reject", "fail to reject"}
+                 Whether H0 can be rejected or not.
+            * "signif" : float
+
+        References
+        ----------
+        .. [1] Lutkepohl, H. 2005. *New Introduction to Multiple Time Series Analysis*. Springer.
+
+        """
+        if not (0 < signif < 1):
+            raise ValueError("signif has to be between 0 and 1")
+
+        allowed_types = (string_types, int)
+
+        if isinstance(caused, allowed_types):
+            caused = [caused]
+        if not all(isinstance(c, allowed_types) for c in caused):
+            raise TypeError("caused has to be of type string or int (or a "
+                            "sequence of these types).")
+        caused_ind = [get_index(self.names, c) for c in caused]
+
+        if causing is not None:
+
+            if isinstance(causing, allowed_types):
+                causing = [causing]
+            if not all(isinstance(c, allowed_types) for c in causing):
+                raise TypeError("causing has to be of type string or int (or "
+                                "a sequence of these types) or None.")
+            causing = [self.names[c] if type(c) == int else c for c in causing]
+            causing_ind = [get_index(self.name
```

Comment:
this is in differences isn't it.
It should mention that it is the short run equation in first differences of endog